airflow/utils/dates.py

```diff
@@ -225,3 +226,14 @@ def days_ago(n, hour=0, minute=0, second=0, microsecond=0):
         second=second,
         microsecond=microsecond)
     return today - timedelta(days=n)
+
+
+def to_iso(execution_date):
+    """ Parse execution date to iso format. """
```

Comment:
Let's add proper pydoc even though some of the other functions in this class do not have it.